packages/core/src/EngineSettings.ts

```diff
@@ -1,9 +1,19 @@
 import { ColorSpace } from "./enums/ColorSpace";
+import { ShadowCascadesMode, ShadowMode, ShadowResolution } from "./shadow";
 
 /**
  * Render settings.
  */
 export interface EngineSettings {
   /** Color space.*/
   colorSpace?: ColorSpace;
+
+  /** How this light casts shadows */
+  shadowMode?: ShadowMode;
+  /** The resolution of the shadow maps. */
+  shadowResolution?: ShadowResolution;
+  /** Number of cascades to use for directional light shadows. */
+  shadowCascades?: ShadowCascadesMode;
+  /** The ratio of cascade distribution. */
+  shadowCascadeSplitRatio?: number;
```

Comment:
`shadowCascadeSplitRatio ` is hard for developer to config and controll.

Use two property `shadowCascade2Split:number`、 `shadowCascade4Split:Vector3`, very easy to config and seperate cache is useful for downgrade

Reference for unity:
<img width="496" alt="image" src="https://user-images.githubusercontent.com/4967545/188544036-8bb056aa-f38b-4d55-8403-62d9565d9b0c.png">